classic_cp/kidneyexchange/IOmanager.jl

```diff
@@ -0,0 +1,33 @@
+function parseInput(raw_input)
+    lines = split(raw_input, '\n')
+    firstLine = split(lines[1], ' ')
+
+    numberOfPairs = parse(Int, firstLine[1])
+    density = parse(Float16, firstLine[2])
+
+    compabilities = Vector{Union{Vector{Int}, Nothing}}(undef, numberOfPairs)
+    fill!(compabilities, nothing)
+
+    for i = 2:numberOfPairs+1
+        if isempty(lines[i])
+            compabilities[i-1] = []
+        else
+            line = split(lines[i],' ')
+            compabilities[i-1] = map(x->parse(Int,x), line)
+        end
+    end
+    return InputData(numberOfPairs, density, compabilities)
+end
```

Comment:
Some comments regarding the input format would be appreciated. Don't forget to add a header to each of your functions to make it easier for future collaborators to work on your code !